openapi/platform/platform.yaml

```diff
@@ -4352,10 +4456,20 @@ components:
       type: object
       description: |-
         [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
-        
+
         An Automation is a trigger and set of actions performed when the trigger is activated.
       x-tags:
         - Automations
+      x-examples:
+        example-1:
+          type: automations
+          id: Automation-91cb048c-c498-43ba-ab99-2ca2d27d2c46
+          attributes:
+            name: Example Automation
+            trigger: /app/account-group/account-group/created
```

Comment:
The example value is `/app/account-group/account-group/created` but the field's docs show that it is an enum of `TaskDefinition-manual-api-order-trigger` or `TaskDefinition-manual-api-account-trigger`. One of them is wrong. We will want to make sure that the value for this attribute matches the value that gets used when filtering.